notebooks/DrizzlePac/Initialization/Initialization.ipynb

```diff
@@ -0,0 +1,319 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class='alert-info'>Before working with HST data, users are advised to consult both the data handbooks and the instrument team websites for the instrument of interest. Additional useful discussion of the drizzling algorithm and of how distortion information is represented in the image header may be found in the <a href=\"http://documents.stsci.edu/hst/HST_overview/documents/DrizzlePac/DrizzlePac.cover.html\">DrizzlePac Handbook</a>.</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Notes about HST files:**\n",
+    "\n",
+    "HST images are stored using the Flexible Image Transport System (FITS). In particular, HST data use a multi-extension FITS (MEF) format. In this format, science data, error arrays, and data quality information for a single observation are stored as different extensions within a single file. The data format for the detector you are working with may be found in your instrument's data handbook, e.g. for [WFC3](http://www.stsci.edu/hst/wfc3/documents/handbooks/currentDHB/wfc3_cover.html) and [ACS](http://www.stsci.edu/hst/acs/documents/handbooks/currentDHB/acs_cover.html).\n",
+    "\n",
+    "\n",
+    "\n",
+    "**Good data quality flags:**\n",
+    "\n",
+    "In April 2018, the ACS instrument team changed the recommended data quality (DQ) flag definitions to consider good when combining dithered observations. The DQ flag 32 is now used to identify unstable pixels (see [ACS ISR 2017-05](http://www.stsci.edu/hst/acs/documents/isrs/isr1705.pdf)). With this change, pixels identified as hot (16) or warm (64) and stable can be treated as good pixels in combined mosaics. Thus, the `driz_sep_bits` and `final_bits` parameters in `astrodrizzle` can be set to 16 + 64 = 80.\n",
+    "\n",
+    "The WFC3 instrument team is planning to implement this change to the DQ flags in early 2019. Users should consult the [WFC3 website](http://www.stsci.edu/hst/wfc3/) for the most up-to-date information about this topic."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Top of Page](#title_ID)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Required Software\n",
+    "\n",
+    "AstroConda is a conda channel that manages astronomy-related Python packages and is maintained at STScI. Users unfamiliar with AstroConda should see the [documentation](https://astroconda.readthedocs.io/en/latest/) for how to install conda and how to select the appropriate software stack. **For most users**, the standard Python 3 software stack that **does not include IRAF/PyRAF** is appropriate and should be used for DrizzlePac. \n",
```

Comment:
"conda channel" is probably a phrase unfamailiar to the average user who comes across this notebook.  I suggest a sentence stating something like "ananconda is a thing to manage python installations.  astroconda adds STScI specific bits. you will need to install anaconda first, and then use astroconda in anaconda <link to the astroconda docs describing this>"

(but then all the rest is good)